api/v1/clustercatalog_types.go

```diff
@@ -54,25 +54,27 @@ const (
 // ClusterCatalog enables users to make File-Based Catalog (FBC) catalog data available to the cluster.
 // For more information on FBC, see https://olm.operatorframework.io/docs/reference/file-based-catalogs/#docs
 type ClusterCatalog struct {
+	// +optional
 	metav1.TypeMeta `json:",inline"`
 
 	// metadata is the standard object's metadata.
 	// More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
+	// +optional
 	metav1.ObjectMeta `json:"metadata"`
 
 	// spec is the desired state of the ClusterCatalog.
 	// spec is required.
 	// The controller will work to ensure that the desired
 	// catalog is unpacked and served over the catalog content HTTP server.
-	// +kubebuilder:validation:Required
+	// +required
```

Comment:
@everettraven I think that is a bug in the linter right?
Why we cannot use `+kubebuilder:validation:Required` ?
Was this option removed?

Reply:
You can configure the linter to use `+kubebuilder:validation:Required`, though it defaults to preferring (and recommending) `+required` instead

Reply:
I don't think an issue needs to be raised anywhere. If you want it to use the `kubebuilder:validation:Required` marker I can configure that.

There is no bug here.